infrateam_first_responder.md

```diff
@@ -1,8 +1,33 @@
-# Guidelines for first responder
-
-The DLSS infrastructure team is trying out a new rotating role of "first responder".  PRs are always welcome, but especially while we're ironing things out and having regular retrospectives about the process.  If it sticks, and/or the access team adopts it, we can get rid of this caveat.
-
-## The "first responder" rotation premises
+# Guide for Infrastructure Team First Responder
+
+The DLSS infrastructure team is using a rotating role of "first responder".  This doc explains a bit about the concept of
+the first responder role and then outlines specific duties and expectations.
+
+PRs to this doc are always welcome, but especially while we're ironing things out and having regular retrospectives about the process.  If it sticks, and/or the access team adopts it, we can get rid of this caveat.
+
+## Table of Contents
```

Comment:
Not a big deal, but would suggest that it's not worth the effort to maintain a TOC.

Reply:
I hear ya.  I'm mostly interested in making this sorta long doc more skimmable and a ToC seemed helpful.  We can ditch the ToC in the future if we want.  Hell, maybe this should be wikis or something.

If this was a google doc, ToC could come along for the ride with little or no maint.

Reply:
A middle-road approach might be to consider using a tool like https://github.com/ekalinin/github-markdown-toc. I've used it in the past to generate the TOC for Sufia and Hyrax documentation. This assumes that we'll have the discipline to regenerate the TOC as appropriate over time, of course, which may be too costly. I could go either way, but thought it worth mentioning as an option.